programs/Makefile

```diff
@@ -131,11 +131,13 @@ mandir      ?= $(datarootdir)/man
 man1dir     ?= $(mandir)/man1
 
 ifneq (,$(filter $(shell uname),OpenBSD FreeBSD NetBSD DragonFly SunOS))
-MANDIR  ?= $(PREFIX)/man/man1
```

Comment:
This patch changes the meaning of `MANDIR` in FreeBSD context.

As I have no experience on the topic, and the [FreeBSD documentation on Man Pages](https://www.freebsd.org/doc/en/books/porters-handbook/makefile-manpages.html) is a bit too short, I have no idea what should be done.

As a consequence, I've retained earlier patch from FreeBSD contributor, which makes `MANDIR` the final destination path of the man page.
This patch would change that.

And I guess there is a name conflict here : `MANDIR` has 2 different meanings, depending on your context, or the FreeBSD context.

Reply:
It's actually possible to change `MANDIR` definition without breaking FreeBSD.
So that part is fine.